Umbraco-Cloud/Set-Up/Manage-CDN-Caching/index.md

```diff
@@ -0,0 +1,128 @@
+In the Purge section of the subpage you can always see how many Purge requests you have available and when.
+
+Please note that the number of Purge requests you have available vary depending on your Cloud Plan. The Plan specific features are listed below.
+
+## Minification Explained
+
+Minification can be enabled for css, javascript and html, and is done through the CDN as part of serving the request. Minification strips whitespace from the css, javascript and html files, so the request is smaller and thus faster to download for the end user.
+
+Please note that using minification through the CDN can have an effect on Content Security Policies, as the files are being changed after they leave the origin. If you are using Content Security Policies we recommend doing minification within the Umbraco site.
+
+## Plan specific features
+
+Access to the different options vary depending on the Umbraco Cloud Plan your project is on.
+Currently the features are available as follows:
+
+- Starter: 
+  - Enable Caching
+  - Cache TTL (see below for minimum TTL)
+  - Minify Css
+  - Minify Javascript
+- Standard: 
+  - Enable Caching
+  - Cache TTL (see below for minimum TTL)
+  - Cache Everything
+  - Minify Css
+  - Minify Javascript
+  - Minify HTML
+- Pro: 
+  - Enable Caching
+  - Cache TTL (see below for minimum TTL)
+  - Cache Everything
+  - Minify Css
+  - Minify Javascript
+  - Minify HTML
+
+The minimum Cache TTL varies as follows:
+
+- Starter: 2 hours / 120 minutes
+- Standard: 30 minutes
+- Pro: 2 minutes
+
+The number of Cache Purge requests within 24 hours: 
+
+- Starter: 2
+- Standard: 10
+- Pro: 20
+
+## CDN Caching and Optimizations subpage
+
+From your Umbraco Cloud Project, click 'CDN Caching & Optimization' from the Settings dropdown to get to the subpage where you can configure the various caching options.
+All settings are scoped per environment, so if you have multiple environments and add your own hostnames to different environments you can select the environment at the top of the page.
+Aside from environments, the 'CDN Caching & Optimization' subpage is divided into two parts: 'Default Settings' and 'Hostname Specific Settings'.
+Use the default settings to configure what should be applied by default to new and existing hostnames.
+
+![Default Settings](images/caching-subpage.png)
+
+If you want to have different caching options for different hostnames, then select the hostname under Hostname specific settings and adjust the options for that specific hostname.
+This might be useful if you want to test the different options on another hostname then your primary hostname.
```

Comment:
```suggestion
If you want to have different caching options for different hostnames, then select the hostname under Hostname specific settings and adjust the options for that specific hostname.
This might be useful if you want to test the different options on another hostname than your primary hostname.
```